Tools/scripts/extract_param_defaults_unittest.py

```diff
@@ -0,0 +1,310 @@
+#!/usr/bin/python3
```

Comment:
could be in Tools/scripts/tests

Reply:
I did that, but the linter does not like it :(

Reply:
Done.